src/router/index.js

```diff
@@ -9,6 +12,41 @@ const routes = [
     path: "/",
     name: "Home",
     component: Home,
+    children: [
+      {
+        path: "list/:listId",
+        name: "ListDetails",
+        component: ListDetails,
+        props: (route) => {
+          const allTasksByListId = store.tasksData.filter(
+            (task) => task.id_list === route.params.listId
+          );
+          return {
+            listName: store.listsData.find(
+              (list) => list.id === route.params.listId
+            ).name,
```

Comment:
The error `name` of `undefined` may come from this statement.

Reply:
Updated!